packages/encoding/test/decodeOnChainData.test.ts

```diff
@@ -5,7 +5,7 @@ import REAL_DECODED from './data/onchain-decoded.json'
 import REAL_DATA from './data/onchain-example.json'
 
 describe('decodeOnChainData', () => {
-  it('decodes real onchain data', () => {
+  it('decodes real onchain data', async () => {
```

Comment:
why?

Reply:
That's on me - forgot to clean this up - removed now.